snap/snapcraft.yaml

```diff
@@ -37,3 +37,4 @@ parts:
     python-version: python3
     source: .
     source-type: git
+    process-dependency-links: true
```

Comment:
can you move the `dependency_links` from `setup.py` to `python-packages:` here?

Reply:
I'll try. Didn't see it was an option.

Reply:
Do we still need `process-dependency-links: true`?